### ISSUE TITLE
Error importing my github workflow repo

### ISSUE THREAD

**Issue by gaow** [bioworkflows/bioworkflows-help#17]
I want to import this repo to bioworkflows.com

https://github.com/gaow/fine-mapping

The interface is intuitive. But I get this error message:

```
   Unsupported archive /app/bioworkflows/media/temp_dir/tmp3s4bubmk
```

Is it how it is supposed to work? Are there special features for a workflow in sync with a github repo like this one? I think to a developer it might make sense that development continues on github but somehow bioworkflows can keep it in sync and add to it something fancier.

**Comment by BoPeng** [comment]
Did you upload a file or provide a URL? If so, which files or URL?

**Comment by BoPeng** [comment]
Right now we look all files for HTML or ipynb, even decompress them, and it should not matter if some files are not recognized.

**Comment by gaow** [comment]
I see ... I provided the URL to the file on github but I should have used `raw` link on github. After doing that, I now get this error:

```
Ooops!!! 500
Looks like something went wrong!
We track these errors automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.
```

and trying to do it again (not refreshing) I get:

```
   Failed to save retrieve source file to workflow gaow/gwas_enrichment/1: duplicate key value violates unique constraint "workflows_attachment_file_name_file_content_2c1605d9_uniq" DETAIL: Key (file_name, file_content)=(gwas_enrichment.ipynb, attachments/fadc394a80cd4ddfcbe86846e68cf81989ae5488986ef65c287441fafe5f99fd) already exists.
```

Here is the link that i want to upload:

https://raw.githubusercontent.com/gaow/fine-mapping/master/workflow/gwas_enrichment.ipynb

Again I think it would be nice if this workflow can be in sync with that on github.

**Comment by BoPeng** [comment]
This seems to be caused by the fact that you already have such a workflow, and you are supposed to go back and "update", instead of creating a new one with the same name. --- but this is something I should improve.

**Comment by BoPeng** [comment]
Let me try to create one.

**Comment by BoPeng** [comment]
I can see the bug, which I will fix soon. :-)

**Comment by BoPeng** [comment]
Fixed. For some reason I had to remove a database and recreate. Its status was ok when poking it.

**Comment by BoPeng** [comment]
I uploaded the cited workflow and put it as public, so you should be able to see it from homepage.

**Comment by gaow** [comment]
Great! I now cleaned up my profile keeping only the successful upload:

https://bioworkflows.com/gaow/torus-enrichment/1

You see the `show source` and `as notebook` don't work. All code cells appear blank.

**Comment by BoPeng** [comment]
I cannot see it because it is private, but that is by design. The error message with clicking that link needs improvement though.

**Comment by BoPeng** [comment]
I should be able to see if you invite me to the workflow, or create a group and invite me to the group. Perhaps you need to upgrade your membership before you can do so, all free though. :-)

**Comment by gaow** [comment]
For the public workflow I think we still have that problem:

https://bioworkflows.com/BoPeng/gwas_enrichment/1

that all code cells appear blank. Is that by design? 

I also tried to add you to the workflow. I didn't have to upgrade my membership at this point. Maybe you received the invitation? But at the end of that Accessibility page it seems the confirmation button for adding you as a maintainer is "confirm fork" when I confirm it I ended up `https://bioworkflows.com/gaow/torus-enrichment/2` not `/1`. This is confusing considering I only want to add you there.
